Xamarin.Essentials/Browser/Browser.mac.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Threading.Tasks;
+using AppKit;
+using Foundation;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Browser
+    {
+        static Task<bool> PlatformOpenAsync(Uri uri, BrowserLaunchOptions options) =>
+            Task.FromResult(NSWorkspace.SharedWorkspace.OpenUrl(new NSUrl(uri.AbsoluteUri)));
```

Comment:
`uri` can be `null` and you'll have a `NullReferenceException`

Reply:
Thanks. The checks happen in the shared code, but we didn't actually check there either.